.changeset/calm-deers-serve.md

```diff
@@ -0,0 +1,23 @@
+---
+'@builder.io/sdk-vue': patch
+---
+
+Feature: add `@builder.io/sdk-vue/node/init` entry point with `initializeNodeRuntime` export that sets the IVM instance.
```

Comment:
- I would describe it in more detail, show how it can be used manually, like https://github.com/BuilderIO/builder/blob/634a0f31f87aca232b719d54111db2578ae3200f/packages/sdks/output/react/CHANGELOG.md?plain=1#L231-L274

and mention the new plugin:

> Exports a Nuxt Plugin `nuxt-initialize-node-runtime-plugin` that initializes `isolated-vm` in node runtimes. Here's how you can use it:

(and show an example)

and then explicitly mention module updates:

> Updated `'@builder.io/sdk-vue/nuxt'` Nuxt module:
> - "added `includeCompiledCss` flag that adds the compiled Builder.io CSS in Nuxt (defaults to `true`)
> - "added `initializeNodeRuntime` flag that executes `nuxt-initialize-node-runtime-plugin` plugin(defaults to `false`)

Reply:
updated, please take a look